README.md

```diff
@@ -970,6 +972,52 @@ If you are using a feature flag with binary variants, the `StatusOverride` prope
 
 In the above example, the feature is enabled by the `AlwaysOn` filter. If the current user is in the calculated percentile range of 10 to 20, then the `On` variant is returned. Otherwise, the `Off` variant is returned and because `StatusOverride` is equal to `Disabled`, the feature will now be considered disabled.
 
+### Variant Service

```

Comment:
I would highlight Dependency Injection here.

Something like

## Use variant services in dependency injection

Reply:
Also, probably makes sense to be in the ToC at the top of the readme.